api/src/main/java/io/grpc/EquivalentAddressGroup.java

```diff
@@ -35,6 +35,12 @@
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/1770")
 public final class EquivalentAddressGroup {
 
+  /**
+   * The authority to be used when constructing Subchannels for this EquivalentAddressGroup.
+   */
+  @Attr
```

Comment:
Add an `@ExperimentalApi` annotation here. Create a new issue to stabilize the API, and include it as the value. You can look at other experimental apis to see the format.

Reply:
Done